hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -74,6 +81,326 @@
       default: "512",
       local: "true"
     },
+
+    // Memory map Info
```

Comment:
@mcy FYI

Reply:
When this is merged, could you file a bug against me to update the OTP DIF? I'd like to do that as a followup after the initial commit.

Reply:
Done #3904